src/components/ProFullCourt/index.tsx

```diff
@@ -30,6 +34,32 @@ const ProFullCourt = () => {
 
   const [size, setSize] = useState({ width: window.innerWidth, height: window.innerHeight });
 
+  const courtAndTileInfo = {
+    beginPointX: (stageMargin - borderLength) * court.courtRatio,
+    beginPointY: (stageMargin - borderLength) * court.courtRatio,
+    endPointX: (stageMargin + courtAreaXLength + borderLength) * court.courtRatio,
+    endPointY: (stageMargin + courtAreaYLength + borderLength) * court.courtRatio,
+    // TO CHANGE LATER: tile size will be passed in instead of hard coding
+    tileSize: 300 * court.courtRatio,
+  };

```

Comment:
Do you think other courts also need this formula for the same calculation? If yes, maybe extract it into a common util file for use on all the other courses.

Reply:
Thank you for this advice. Other courts would need calculation but with different formulas, as this ProFullCourt has a border that other courts do not have. In a future card, the calculation for other courts (if same), will be extracted to a util file.